astroid/modutils.py

```diff
@@ -101,26 +102,15 @@
             pass
 
 if platform.python_implementation() == "PyPy":
-    # The get_python_lib(standard_lib=True) function does not give valid
-    # result with pypy in a virtualenv.
-    # In a virtual environment, with CPython implementation the call to this function returns a path toward
-    # the binary (its libraries) which has been used to create the virtual environment.
-    # Not with pypy implementation.
-    # The only way to retrieve such information is to use the sys.base_prefix hint.
-    # It's worth noticing that under CPython implementation the return values of
-    # get_python_lib(standard_lib=True) and get_python_lib(santdard_lib=True, prefix=sys.base_prefix)
-    # are the same.
-    # In the lines above, we could have replace the call to get_python_lib(standard=True)
-    # with the one using prefix=sys.base_prefix but we prefer modifying only what deals with pypy.
-    STD_LIB_DIRS.add(get_python_lib(standard_lib=True, prefix=sys.base_prefix))
-    _root = os.path.join(sys.prefix, "lib_pypy")
-    STD_LIB_DIRS.add(_root)
-    try:
-        # real_prefix is defined when running inside virtualenv.
-        STD_LIB_DIRS.add(os.path.join(sys.base_prefix, "lib_pypy"))
-    except AttributeError:
-        pass
-    del _root
+    # PyPy stores the stdlib in two places: sys.prefix/lib_pypy and sys.prefix/lib-python/3
+    # sysconfig.get_path on PyPy only returns the first so we patch this manually.
+    # Note that sysconfig.get_path expands: '{installed_base}/lib-{implementation_lower}'
+    # sys.prefix/lib-pypy resolves into sys.prefix/lib_pypy
+    STD_LIB_DIRS.add(sysconfig.get_path("stdlib"))
```

Comment:
`sysconfig.get_path("stdlib")` is already part of the set -> #1323.
Instead we should include `lib_pypy` here. Maybe something like this?
```py
from pathlib import Path

Path(sysconfig.get_path("stdlib").parent.joinpath("lib_pypy"))
```

Furthermore, we should probably do the same for `platstdlib`.